### PR TITLE
Added new created_at tag to AWS resources

### PR DESCRIPTION
**Pull request by blarghmatey**
Updated the tag assignments to consistently set the `business_unit`
and `created_at` tags for AWS resources

### CODE REVIEW

File: salt/orchestrate/aws/bootcamp-ecommerce/rds.sls [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% from "orchestrate/aws_env_macro.jinja" import VPC_NAME, VPC_RESOURCE_SUFFIX,
  ENVIRONMENT, subnet_ids with context %}
-
+{% set ISOFORMAT = '%Y-%m-%dT%H:%M:%S' %}
```
Comment (shaidar) [inline salt/orchestrate/aws/bootcamp-ecommerce/rds.sls]: Should this be 'ISO8601'?

File: salt/orchestrate/aws/map_templates/instance_map.yml [hunk 2]
```diff
@@ -10,10 +12,12 @@
       tag:
         environment: {{ environment_name }}
         role: {{ service_name }}
+        created_at: {{ salt.status.time(format=ISO8601) }}
```
Comment (shaidar) [inline salt/orchestrate/aws/map_templates/instance_map.yml]: AWS has 'Launch Time' for EC2 instances, so wouldn't this be redundant or is the idea to have the same value across all objects in AWS?

Comment (blarghmatey) [inline salt/orchestrate/aws/map_templates/instance_map.yml]: It is redundant for EC2, but most other resources are lacking that. This is just to create a consistent set of values that we can refer to without having to look in different places for each type of asset.

File: salt/orchestrate/aws/bootcamp-ecommerce/rds.sls (hunk 1 above)
Comment (blarghmatey) [inline salt/orchestrate/aws/bootcamp-ecommerce/rds.sls]: Yes :smile:

**Comment by shaidar** [comment]
:thumbsup:

**Pull request merged**